app/components/language_toogle/LanguagesToogle.jsx

```diff
@@ -0,0 +1,26 @@
+import { useRouter } from "next/router"
+import Link from "next/link"
+import languageDetector from "../../../lib/languageDetector"
+
+export default function LanguagesToogle({locale,...rest}) {
```

Comment:
1. currently on the project we are setting the export default at the end of the component
2. this component name is too much similar of "LanguageToogle" component, could we change "LanguagesToogle" to another name?

Reply:
I understand you, It was resolved

Reply:
Ok, It was resolved